File: songs/erlang/my_songs.erl
-module(my_songs).
-export([songs/0]).

-record(song,  { title, tags = [] }).
% -type song() :: #song{}.
% -export_type([song/0]). 

songs() -> [
  #song{ title = "Cantaloupe Island",
         tags  = ["60s", "jazz"] },
  #song{ title = "Let it Be",
         tags  = ["60s", "rock"] },
  #song{ title = "Knockin' on Heaven's Door",
         tags  = ["70s", "rock"] },
  #song{ title = "Emotion",
         tags  = ["70s", "pop"] },
  #song{ title = "The River"}
].

File: songs/scala/MySongs.scala
package mysongs

case class Song(
  title: String,
  tags: Option[List[String]]
)

object MySongs {
  val songs: List[Song] = List(
    Song( "Cantaloupe Island",
          Some(List("60s", "jazz")) ),
    Song( "Let It Be",
          Some(List("60s", "rock")) ),
    Song( "Knockin' on Heaven's Door",
          Some(List("70s", "rock")) ),
    Song( "Emotion",
          Some(List("70s", "pop")) ),
    Song( "The River", None )
  )
}
